src/v2/BaseLoginRouter.js

```diff
@@ -93,7 +93,7 @@ export default Router.extend({
   },
 
   handleIdxResponseFailure (error = {}) {
-    if (error && error.details && error.details.stateHandle) {
+    if (error?.details?.stateHandle) {
```

Comment:
is it going to be supported in all browsers ?

Reply:
It transpiled to JavaScript that any browser would understand. kind of same as we write `a && a.b && a.b.c` but in slim fashion.
(I guess it's because we didn't set tranpile target to be only browsers that already support chain operator)

Reply:
Yes, it's dictated by our browserslist config in package.json - 
```
"browserslist": [
  "> 0.1%",
  "not ie <= 9"
],
```
By changing the above config and transpiling again resulted in a considerable size difference in `okta-sign-in.js`

Reply:
@anipendakur-okta curious what you changed and the size difference?

Reply:
Hi @haishengwu-okta I meant I changed the browserslist config in package.json to this and noticed a change -
```
  "browserslist": [
    "last 5 chrome version"
  ]
```
![image](https://user-images.githubusercontent.com/62976351/88995538-c67dfd00-d29f-11ea-97c2-999c3253e77a.png)